toolz/tests/test_curried.py

```diff
@@ -38,3 +38,19 @@ def test_reduce():
 
 def test_module_name():
     assert toolz.curried.__name__ == 'toolz.curried'
+
+
+def test_curried_operator():
```

Comment:
Can we add a test that verifies the `toolz.curried.operator` namespace is populated?  Currently, if you do `cop.add`, this PR will fail in Python 2.